community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/executionplan/CompiledExecutionResult.scala

```diff
@@ -271,5 +55,9 @@ class CompiledExecutionResult(taskCloser: TaskCloser,
 
     compiledCode.executionPlanDescription()
   }
-}
 
+  override def queryStatistics() = InternalQueryStatistics()
+
+  //TODO delegate to compiled code once writes are being implemented
+  override def executionType = READ_ONLY
```

Comment:
In this case you are commenting on `CompiledExecutionResult` it doesn't support write operations yet, when we do we should fix update this